### PR TITLE
Convert server/startup/migrations to JS

### PR DESCRIPTION
**Pull request by rodrigok**
@RocketChat/core

### CODE REVIEW

File: server/startup/migrations/v020.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+				'file._id': 1
+			}
+		};
+
+		const cursorFileMessages = RocketChat.models.Messages.find(msgQuery, msgOptions);
+		if (!cursorFileMessages.count()) {
+			return;
+		}
+
+		cursorFileMessages.fetch().each((msg) => {
```
Comment (graywolf336) [inline server/startup/migrations/v020.js]: Should this be `forEach`?

File: server/startup/migrations/v033.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
+RocketChat.Migrations.add({
+	version: 33,
+	up() {
+		const scriptAlert = '/**\n * This scrit is out-of-date, convert to the new format\n * (https://github.com/RocketChat/Rocket.Chat/wiki/WebHook-Scripting)\n**/\n\n';
```
Comment (graywolf336) [inline server/startup/migrations/v033.js]: Fix the misspelling `scrit` to `script`. And change the url to: https://rocket.chat/docs/administrator-guides/integrations

File: server/startup/migrations/v020.js (hunk 1 above)
Comment (rodrigok) [inline server/startup/migrations/v020.js]: Thanks, done!

File: server/startup/migrations/v033.js (hunk 2 above)
Comment (rodrigok) [inline server/startup/migrations/v033.js]: Thanks, done!

**Pull request merged**